dotnet/ChatSub.cs

```diff
@@ -0,0 +1,6 @@
+channel.Presence.Subscribe(
+    PresenceAction.Enter, 
+    message => UI.ShowMemberEntered(message));
+
+channel.Subscribe(message => {
```

Comment:
we can remove `{` and `}` to make it look better

Reply:
Yeah, I think we also need proper code formatting across all snippets, currently it feels non-readable

Reply:
As much as possible I've tried to stay within the budget of 5 lines and 64 columns.

Reply:
Resolving this conversation so that we can land this work as it's needed for other teams to work on tomorrow.